assets/scripts/app/SkyObjects.jsx

```diff
@@ -55,7 +55,7 @@ function SkyObjects ({ objects = [] }) {
         // TODO: Render placeholder ("dev") asset? That way it will be more
```

Comment:
@louh is this to-do worth doing right now? Should it be part of this PR or a separate issue?

Reply:
It's probably not worth doing for this PR. But we can turn make it into an issue.